xml/System.Net.Mail/MailMessage.xml

```diff
@@ -938,6 +947,7 @@
   
  [!code-cpp[NclMailSync#10](~/samples/snippets/cpp/VS_Snippets_Remoting/NCLMailSync/CPP/NclMailSync.cpp#10)]
  [!code-csharp[NclMailSync#10](~/samples/snippets/csharp/VS_Snippets_Remoting/NCLMailSync/CS/mail.cs#10)]  
+ [!code-vb[NclMailSync#10](~/samples/snippets/visualbasic/VS_Snippets_Remoting/NCLMailSync/vb/mail.vb#10)]  
```

Comment:
I don't see where the From property is explicitly set, but this is not unique to the snippet you added.